Ednpoints.txt

```diff
@@ -108,25 +111,37 @@ DELETE: /dishes/:dishId - delete a particular dish
 GET: /orders - get all orders info
 GET: /orders/:orderId - get a particular order
 PATCH: /orders/:orderId - update a particular order
-DELETE: /dishes/:orderId - delete a particular order
-
-GET: /reviews?dish=dishId&chef=chefId - get comments of a particular dish
-PATCH: /reviews/:reviewId - update a particular comment
-DELETE: /reviews/:reviewId - delete a particular comment
+DELETE: /order/:orderId - delete a particular order
 
-GET: /notifications/:userId - get notifications of this chef
-POST: /notifications - create new notification
+**notifications will not be saved on the server
+POST: /notifications - create new notification (messages about technical work, policy updates, new features etc.)
 
 GET: /orders/stats - get statistics about orders
 GET: /chefs/stats - get statistics about chefs
-GET: /deliveries/stats - get statistics about delivery persons
+GET: /couriers/stats - get statistics about delivery persons
 GET: /users/stats - get statistics about users
 GET: /revenue/stats - calculate how much was purchased through the app
 
 /************/
 
-ПИТАННЯ:
+REVIEW ENDPOINTS:
+
+There are two ways how we can show reviews (reviews are available for users, chefs and admins (to delete the comment)). Which one would you recommend:
+1. As reviews array of the dish document
+GET: /dishes/:dishId/reviews - get comments to this dish
+POST: /dishes/:dishId/reviews - update person's comment to this dish
+PATCH: /dishes/:dishId/reviews/:reviewId - update person's comment to this dish
+DELETE: /dishes/:dishId/reviews/:reviewId - delete person's comment to this dish
+
+2. As a separate collection
+GET: /reviews/by-chef/:chefId - get all comments related to a particular chef's dishes
+GET: /reviews/by-dish/:dishId - get all comments of the dishes
+POST: /reviews - create a new comment to this dish
+PATCH: /reviews/:reviewId - update person's comment to this dish
+DELETE: /reviews/:reviewId - delete person's comment to this dish
+
+/************/
 
-1) Яким способом нам краще здійснити оплату і відображати баланс? Чи буде це просто заповнення полів із фіктивним переказом коштів, чи реальне під'єднання платіжної системи з використанням якихось ось тестових карток, чи якийсь інший варіант?
+QUESTIONS
 
-2) Яка інфа потрібна і може бути збережена на сервері щодо транзакцій, доходу, поточного балансу тощо?
+The issue regarding payment and implementation of money transfers is a difficult one for us due to the lack of experience in this topic among the whole team, so it would be great to have a meeting with mentors regarding this issue
```

Comment:
here the link to Liqpay docs, they can provide payment separation for few accounts
https://www.liqpay.ua/documentation/api/aquiring/checkout/doc#:~:text=%D0%9F%D0%BB%D0%B0%D1%82%D1%96%D0%B6%20%D0%B7%20%D1%80%D0%BE%D0%B7%D1%89%D0%B5%D0%BF%D0%BB%D0%B5%D0%BD%D0%BD%D1%8F%D0%BC,JSON%20%D1%80%D1%8F%D0%B4%D0%BA%D0%B0%3A

we can split payment, but to do so we need to create Liqpay account for each side (chef and courier) automatically after registration
or we can receive money on our Liqpay company and then "send money to chefs and couriers upon their request from the app interface"

first approach more complicated, but fully automated, we don't need to think about payment requests
second approach is the one if first won't work for us